src/mcode-pilot/utils/recordToProps.js

```diff
@@ -1,9 +1,13 @@
 import _ from 'lodash';
 import FluxTumorDimensions from '../../model/tumor/FluxTumorDimensions';
+import FluxTNMClinicalPrimaryTumorClassification from '../../model/oncocore/FluxTNMClinicalPrimaryTumorClassification';
+import FluxTNMClinicalRegionalNodesClassification from '../../model/oncocore/FluxTNMClinicalRegionalNodesClassification';
+import FluxTNMClinicalDistantMetastasesClassification from '../../model/oncocore/FluxTNMClinicalDistantMetastasesClassification';
 
 export default function getProps(patient, condition) {
 
     const tumorMarkers = patient.getMostRecentTumorMarkers(condition);
+    const tnminfo = processPanel(condition.getMostRecentClinicalStaging()._tnmStageGroup._panelMembers.observation, patient);
```

Comment:
If there isn't a most recent clinical staging this line is going to cause runtime errors